services/catalog/src/simcore_service_catalog/api/rest/_services_extras.py

```diff
@@ -0,0 +1,18 @@
+from typing import Annotated, Any
+
+from fastapi import APIRouter, Depends
+from models_library.services import ServiceKey, ServiceVersion
+
+from ...services.director import DirectorApi
+from ..dependencies.director import get_director_api
+
+router = APIRouter()
+
+
+@router.get("/{service_key:path}/{service_version}/extras")
+async def get_service_extras(
+    service_key: ServiceKey,
+    service_version: ServiceVersion,
+    director_client: Annotated[DirectorApi, Depends(get_director_api)],
+) -> dict[str, Any]:
```

Comment:
is it completely unstructured? Should we start creating a proper model for extras?

Reply:
also think we could start using validated models on both ends.
@pcrespov, that is also valid for the service details...